wiki/Modding/Kudosu/en.md

```diff
@@ -0,0 +1,7 @@
+# Kudosu
+
+**Kudosu** is a reward system for [Modding](/wiki/Modding). It awards points for successful modposts on beatmap discussion <!-- TODO: beatmap discussion stub -->pages. Accumulated points can be seen on a user's profile-page.
+
+In Modding v1 <!-- TODO: modding v1 stub -->, Kudosu was exchangeable for Star Priority. <!-- TODO: Star Priority stub? -->
```

Comment:
kudosu and star priority aren't proper nouns, star priority can link to `/wiki/Glossary#star-priority` (you could keep the TODO comment if you want a reminder to go back and replace this)

Reply:
Additionally, I think it might be a good idea to use one big TODO comment at the bottom of an article listing potential links instead. The way you organize it currently makes it a bit messy. Something like:

> TODO: Add links beatmap discussion, forum discussion, star priority, etc